### PR TITLE
Adding ButterCMS Starter

### PR DESCRIPTION
**Pull request by jakelumetta**
We're big fans of Gridsome and Vue so we're excited to launch this Starter project that utilizes our new Plugin (https://gridsome.org/plugins/gridsome-source-buttercms) for CMS-powered marketing page + blog app.

Looking forward to any feedback / questions.

We'll be launching a better guide for our plugin very soon as well.

### CODE REVIEW

**Comment by jakelumetta** [comment]
Hi @tomtev - who's the best person to tag for reviewing this type of PR?  Thanks much!

**Comment by tomtev** [comment]
Hi :) 

These does not seem to work:
repo: ButterCMS/gridsome-starter-buttercms
preview: https://buttercms-gridsome-starter.netlify.com/

That repo on Github also has node_modules and does not look like a Gridsome project.

**Comment by jakelumetta** [comment]
Hey @tomtev - Sorry about that.  We've corrected both the repo and the starter project:  https://buttercms-gridsome-starter.netlify.com/

**Comment by jakelumetta** [comment]
Hi - @tomtev 

Is this looking good to you now? Thanks!

**Comment by jakelumetta** [comment]
Hi @hjvedvik - any chance you could take a look at this?  We're (ButterCMS) looking really ramp up our support of Gridsome.

**Comment by tomtev** [comment]
Still not linking to correct starter 😅 Need to be a complete Github url to the starter

**Comment by jakelumetta** [comment]
Are you referring to these URLs?  https://share.getcloudapp.com/2NuB8pBn

**Comment by jakelumetta** [comment]
Hi @tomtev - just following up. Would love to get this merged in.

File: src/components/home/HomeHowItWorkSimple.vue [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
       <Card title="Bring your data">
         <h4>CMSs</h4>
-        <p>Any Headless CMS, Contentful, WordPress, Drupal, Sanity.io, etc.</p>
+        <p>Any Headless CMS, <a href="https://buttercms.com">ButterCMS</a>, Contentful, WordPress, Drupal, Sanity.io, etc.</p>
```
Comment (mtlynch) [inline src/components/home/HomeHowItWorkSimple.vue:7]: Can we remove the link here? There are lots of companies mentioned on this page, so we should either be linking all of them or none of them.

File: src/components/home/HomeHowItWorks.vue [hunk 2]
```diff
@@ -21,7 +21,7 @@
 
         <Card style="border-radius: 0">
           <h4>CMSs</h4>
-          <p>Any Headless CMS, Contentful, WordPress, Drupal, DatoCMS, Sanity.io, GraphCMS etc.</p>
+          <p>Any Headless CMS, <a href="https://buttercms.com">ButterCMS</a>, Contentful, WordPress, Drupal, DatoCMS, Sanity.io, GraphCMS etc.</p>
```
Comment (mtlynch) [inline src/components/home/HomeHowItWorks.vue:24]: Ditto about linking here.

**Review by mtlynch — CHANGES_REQUESTED** [review]
Thanks for submitting, @jakelumetta!

I checked out the repo and it still seems to be incomplete as a starter. I filed some bugs on the things that prevent it from being complete:

https://github.com/ButterCMS/gridsome-starter-buttercms/issues/created_by/mtlynch

~~If you address those issues, I can merge this in.~~

### Update (2020-09-09)

I am no longer willing to review this PR.

I don't feel that the ButterCMS team is practicing due diligence in what they submit for review. Simple bugs I've filed have required months of back-and-forths. I went into more detail on [this issue comment](https://github.com/ButterCMS/gridsome-starter-buttercms/issues/9#issuecomment-689879793).

I do not speak on behalf of the Gridsome team, so someone else might be willing to review this and help you get it merged in.

**Comment by jonalecpalustre** [comment]
We've taken care of the other issues for our Gridsome starter here - https://github.com/ButterCMS/gridsome-starter-buttercms/issues?q=is%3Aissue+author%3Amtlynch+is%3Aclosed

Also, remove links that was requested by Michael above.

**Comment by jonalecpalustre** [comment]
@tomtev Would you be able to assist on reviewing and merging this PR please? Thank you!

**Comment by jakelumetta** [comment]
@tomtev  @mtlynch Happy New Year.  Any chance we can get a final review here?

**Pull request closed without merging**